react/PWAContext.js

```diff
@@ -15,9 +15,13 @@ const PWAProvider = ({ settings, children }) => {
 
   useEffect(() => {
     const handleBeforeInstall = e => {
-      const { promptOnCustomEvent } = settings
-      if (promptOnCustomEvent && !captured.current) {
+      const { addToHomeScreenPrompt } = settings
+      if (addToHomeScreenPrompt !== "default" && !captured.current) {
         e.preventDefault()
+        
+        if(addToHomeScreenPrompt === "disable")
```

Comment:
please always insert `if (x) {}` to prevent issues 😬

Reply:
Done